gce-wake.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Start all the instances in the specified GCE deployment
+  - hosts: localhost
+    tasks:
+      - name: Deploy gce.py
+        file: src=gce.py dest=./gce.py mode=0744
+      - name: Deploy gce_account.pem
+        file: src=gce_account.pem dest=./gce_account.pem mode=0600
```

Comment:
I'm copying from localhost (current directory - thus ansible-terraform dir) to the localhost temp directory where the script runs (/tmp/ansible23423433). Otherwise I'd have to specify full path. 
Also, given that the full path is already in secrets.py, I can probably remove copying pem/json altogether...